packages/packages/commonwell-sdk/src/models/commonwell.ts

```diff
@@ -0,0 +1,125 @@
+import * as fs from "fs";
+import { RequestMetadata } from "../client/commonwell";
+import { CertificateParam, CertificateResp } from "./certificates";
+import { DocumentQueryResponse } from "./document";
+import { Identifier, StrongId } from "./identifier";
+import { NetworkLink, PatientLinkProxy } from "./link";
+import { Organization, OrganizationList } from "./organization";
+import { Patient, PatientNetworkLinkResp, PatientSearchResp, PatientLinkResp } from "./patient";
+import { PatientLink, PatientLinkSearchResp, Person, PersonSearchResp } from "./person";
+
+export interface CommonWellType {
```

Comment:
Not a huge deal, but I suggest we rename this to `CommonWellAPI` (it matches the `makeCommonWellAPI` and it conveys what it means), rename the file to the same name, and move it to `../client` since its related to the client and not the models.

Reply:
Good call - was going to comment something similar in my previous review, but the comment got lost some how 🤷